docs/sources/basics/exemplars/view-exemplars.md

```diff
@@ -0,0 +1,63 @@
++++
+title = "View exemplar data"
+description = "Exemplars"
+keywords = ["grafana", "concepts", "exemplars", "prometheus"]
+weight = 400
++++
+
+# View exemplar data
+
+When support for exemplar support is enabled for a Prometheus data source, you can view exemplar data either in the Explore view or from the Loki log details. 
```

Comment:
By Loki log details, do you mean in Loki or by viewing Loki logs in Explore? Unclear.

Reply:
Also, I suggest linking to the exemplar concept topic in the intro, in case the reader got here from a web search and might have missed information they would have gotten by browsing.

Reply:
> By Loki log details, do you mean in Loki or by viewing Loki logs in Explore? Unclear.

Loki logs in Explore